src/requireCssModule.js

```diff
@@ -18,19 +18,57 @@ import type {
   StyleModuleMapType
 } from './types';
 
-const getTokens = (runner, cssSourceFilePath: string, filetypes): StyleModuleMapType => {
+type FileTypeOptions = {|
+  syntax: string,
+  plugins: Array<string>
+|};
+
+const getFiletypeOptions = (cssSourceFilePath: string, filetypes: Object): ?(string|FileTypeOptions) => {
```

Comment:
> Where does the string come from, though?

I see.

Lets remove the shortcut. This is going to be a breaking change.

Require that `FileTypeOptions` is always present.